testing/run_tests.py

```diff
@@ -370,17 +370,19 @@ def RunObjcTests(ios_variant='ios_debug_sim_unopt'):
   ios_out_dir = os.path.join(out_dir, ios_variant)
   EnsureIosTestsAreBuilt(ios_out_dir)
 
-  pretty = "cat" if subprocess.call(["which", "xcpretty"]) else "xcpretty"
   ios_unit_test_dir = os.path.join(buildroot_dir, 'flutter', 'testing', 'ios', 'IosUnitTests')
 
+  # Avoid using xcpretty unless the following can be addressed:
+  # - Make sure all relevant failure output is printed on a failure.
+  # - Make sure that a failing exit code is set for CI.
+  # See https://github.com/flutter/flutter/issues/63742
```

Comment:
Should we even mention this?  I never want to hide the full Xcode build results when there's an error, it's frustrating when I can't reproduce the failure locally.  `xcpretty` is nice for end users, not me who wants to see all the clang flags and search paths.  "relevant failure output" is pretty different depending on what's failing.

Reply:
I did that thinking that a year or two from now, someone is going to say "Hey, we should add xcpretty to make this easier to read!" again :)

Reply:
And it'll probably be you or me. 😄